### PR TITLE
nodes: rename Render* to Draw*

### PR DESCRIPTION
**Pull request by mbouron**
RenderToTexture is not renamed as it is not a drawing node.

### CODE REVIEW

**Review by ubitux — COMMENTED** [review]
Doc needs some more adjustments:
```
doc/usr/expl/media-time-remapping.md:sub-tree (in our case, it will typically be a [Render] using a [Media] node as
doc/usr/expl/media-time-remapping.md:[Render]: /usr/ref/libnopegl.md#render
doc/usr/expl/noise.md:attached to the associated [Render] or [Compute].
doc/usr/expl/noise.md:[Render]: /usr/ref/libnopegl.md#render
doc/usr/expl/scopes.md:can be attached to [Render] vertex or fragment resources, or [Compute]
doc/usr/expl/scopes.md:[Render]: /usr/ref/libnopegl.md#render
doc/usr/expl/shaders.md:When constructing a graphic pipeline (`Render` node), the geometry information
doc/usr/expl/shaders.md:`Render.attributes` or `Render.instance_attributes`. These attributes will be
doc/usr/expl/shaders.md:`Render.vert_resources`. Since this parameter is a dictionary, the user can
doc/usr/expl/shaders.md:`Render.frag_resources`. Since this parameter is a dictionary, the user can
doc/usr/expl/shaders.md:`Render.vert_resources`, `Render.frag_resources` or `Compute.resources` to be
doc/usr/expl/shaders.md:render = Render(geometry, program)
doc/usr/howto/c-api.md:and how a basic `Render` node works. If not, you are encouraged to check out
doc/usr/howto/path.md:Rendering a path with a single cubic curve looks like this:
doc/usr/howto/shapes.md:Rendering a triangle
doc/usr/howto/shapes.md:Rendering a quadrilateral
doc/usr/howto/shapes.md:Rendering a circle with 64 points
doc/usr/howto/shapes.md:Rendering a custom geometry composed of 4 random points
doc/usr/ref/pynopegl.md:`*Dict`                 | `dict`                              | `render = Render(geometry, frag_resources={'tex0': t0, 'tex1': t1})`
```

**Comment by ubitux** [comment]
You might also want to `git ls-files|xargs sed -i 's/render =/draw =/'`

File: tests/benchmark.py [hunk 1]
```diff
@@ -312,23 +312,23 @@ def _get_random_layer(cfg: ngl.SceneCfg, rng, t0, t1, enable_computes, layer=4):
...
             )
-            rtt_render = _get_random_transform(rng, t0, t1, rtt_render)
+            rtt_draw = _get_random_transform(rng, t0, t1, rtt_draw)
             t_start, t_end = _get_random_time_range(rng, t0, t1)
-            rtt_group = ngl.Group(children=(rtt, rtt_render))
+            rtt_group = ngl.Group(children=(rtt, rtt_draw))
             t_filter = ngl.TimeRangeFilter(rtt_group, t_start, t_end)
 
-            # Draw both the children and the rendered texture
+            # Draw both the children and the drawed texture
```
Comment (ubitux) [inline tests/benchmark.py:325]: drawn

Comment (mbouron) [reply tests/benchmark.py]: Kept `renderered`, it makes reference to the rtt.

File: tests/refs/scope_draw_histogram_luma_only.ref [hunk 2]
```diff
@@ -0,0 +1,5 @@
+549D75DD35D17DDC39917DD428817DD5 549D75DD35D17DDC39917DD428817DD5 549D75DD35D17DDC39917DD428817DD5 00000000000000000000000000000000
+553D55D1559D4DDC3D917DD428817DD5 553D55D1559D4DDC3D917DD428817DD5 553D55D1559D4DDC3D917DD428817DD5 00000000000000000000000000000000
+54B575D545D57DD439957DD028817DD5 54B575D545D57DD439957DD028817DD5 54B575D545D57DD439957DD028817DD5 00000000000000000000000000000000
+553D55F555914DDC3D916DD468807DD7 553D55F555914DDC3D916DD468807DD7 553D55F555914DDC3D916DD468807DD7 00000000000000000000000000000000
+55DD55D575916DDC39917DD428817DD5 55DD55D575916DDC39917DD428817DD5 55DD55D575916DDC39917DD428817DD5 00000000000000000000000000000000
```
Comment (ubitux) [inline tests/refs/scope_draw_histogram_luma_only.ref:5]: You need to remove `tests/refs/scope_render_histogram_luma_only.ref`

Comment (mbouron) [reply tests/refs/scope_draw_histogram_luma_only.ref:5]: Done.

**Review by ubitux — APPROVED** [review]
LGTM; please update the `next` branch in the `nope.demos` repository though

Edit: most `pynopegl-utils/pynopegl_utils/examples` scenes are broken and need adjustments

**Review by ubitux — CHANGES_REQUESTED** [review]
Most `pynopegl-utils/pynopegl_utils/examples` scenes are broken and need adjustments

**Review by ubitux — APPROVED** [review]
LGTM, see my comment on the `nope.demos` repository though

**Pull request merged**